### ISSUE TITLE
popover styling is incorrect

### ISSUE THREAD

**Issue by cormacmccarthy** [ExactTarget/fuelux-mctheme#183]
They should look like this:

![image](https://cloud.githubusercontent.com/assets/8006971/6473724/3e93ee6e-c1c9-11e4-89f2-df7dc31702a6.png)

**Comment by chriscorwin** [comment]
Heya @dkilgore and or @cormacmccarthy : 

Is this just plain old bootstrap code? Or is there a FUX control or widget or doohickey that I'm not seeing?

**Comment by cormacmccarthy** [comment]
I believe this is Bootstrap

**Comment by chriscorwin** [comment]
I wonder why it's not in our example HTML then?

I'll fix _that_ too.

**Comment by chriscorwin** [comment]
Which of these colors do you suppose the `.popover-heading` should be?

![screenshot 2015-03-20 13 22 29](https://cloud.githubusercontent.com/assets/362417/6757024/44b4b60a-cf04-11e4-89f0-733eaa90e043.png)


The Modal uses a non-prescribed colour:

```less
	.modal-header {
		background: #eaf0f3;
		border-radius: 5px 5px 0 0;
	}
```

I know that @interactivellama and perhaps @jamin-hall  (?) did some colours work, and perhaps that's already been caught, but... nonetheless, I'd like to get it right.

**Comment by chriscorwin** [comment]
Here's what I've got so far:

![screenshot 2015-03-20 15 06 12](https://cloud.githubusercontent.com/assets/362417/6758975/b5f3bf42-cf12-11e4-9401-379f1c27dbef.png)

Considering the lack of guidance in the Style Guide as to actual colours for this stuff, I went with border colours that are less harsh than what was in your example image.

I made the popovers match modals, basically, though I did update modals.less to have cleaner code and colours that match the style guide better.

@mbeard and @futuremint and @swilliamset ... thoughts?
